cdc_stream/main.cc

```diff
@@ -49,8 +50,10 @@ Usage: cdc_stream [flags] start windows_dir [user@]host:linux_dir
        Streams the Windows directory windows_dir to directory linux_dir on the
        Linux host using SSH username user.
  
-       cdc_stream [flags] stop host
-       Stops the streaming session to the given Linux host.)!";
+       cdc_stream [flags] stop [user@]host:linux_dir
```

Comment:
Suggestion for future work: Make stop work without host/dir arg when there's only one session.

Reply:
Thanks for the suggestion. Probably wouldn't be hard to implement.